origin-bridge/src/logger.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const Logger = require('logplease')
+Logger.setLogLevel('DEBUG')
```

Comment:
Wonder if we should log level by default to INFO rather than DEBUG ?

Reply:
Will do!